packages/compass/src/app/styles/palette.less

```diff
@@ -1,151 +1,46 @@
-@sidebar-bg: @slate1;
-@sidebar-width: 250px;
-
-/* MongoDB Color Palette */
-/* commented colors have been deprecated */
-@green0: #0E7E3D; 
-@green1: #168B46;
-@green2: #13AA52; // Primary green
-@green3: #1CC061; 
-@green4: #89E5B3;
-@green5: #CCFFE1; 
-@green6: #EFFEF6;
-
-@gray0: #313030;
-@gray1: #494747;
-@gray2: #615f5f;
-@gray3: #807f7f;
-@gray4: #a09f9e;
-@gray5: #bfbfbe;
-
-/* UI Grays */
-@gray6: #dee0e3;
-@gray7: #ebebed;
-@gray8: #f5f6f7;
-@gray9: #AAAAAA;
-@gray10: #E4E4E4;
-
-@slate0: #42494f;
-@slate1: #4c5259;
-@slate2: #70757a;
-
-@blue3: #5b81a9;
-@blue4: #84a1bf;
-@blue5: #adc0d4;
-
-@purple0: #373e50;
-@purple1: #535d77;
-@purple2: #6e7c9f;
-@purple3: #8b96b2;
-@purple4: #a8b0c5;
-@purple5: #c5cbd9;
-
-/* Other Colors */
-@red: #ed271c;
-@lightRed: #ffeaea;
-@yellow: @alertOrange;
-@pw: #fff;
-@cautionOrange: @alertOrange;
-@errorBackground: #fdd0d1;
-@greenBg: @green6;
-
-/* Text Colors */
-/* default text color is @gray1 */
-@linkText: #006cbc;
-@linkOnDark: #2898dd;
-@warningText: #ffb618;
-@successText: @green2;
-@errorText: #ed271c;
-@mutedText: @gray4;
-@veryMutedText: @gray5;
-@subNavText: #6f6763;
-
-/*Chart Colors*/
-@chart0: #F68A1E;
-@chart1: #43B1E5;
-@chart2: #F38183;
-@chart3: #26A348;
-@chart4: #FBB129;
-@chart5: #92A83B;
-@chart6: #46929A;
-@chart7: #D381B3;
-@chart8: #85CA98;
-
-@chartAverage: #d29e29;
-@chartTotal: #6ba442;
-
-@chartRestart: #ef4c65;
-@chartBecamePrimary: #fbb129;
-@chartBecameSecondary: #7f6a4e;
-
-/* Alert Colors */
-@alertOrange: #fbb129;
-@alertOrangeBg: #fff4dc;
-@alertOrangeBorder: #a06c29;
-@alertRed: #ef4c4c;
-@alertRedBg: #feeded;
-@alertRedText: #A94442;
-@alertBlue: #43b1e5;
-
-/* Semantic Colors */
-
-@inactiveIcon: @gray4;
-
-/* MongoDB Color Palette 2019 */
-@black: #061621;
-@grayDark3: #21313C;
-@grayDark2: #3D4F58;
-@grayDark1: #5D6C74;
-@grayBase: #9FA1A2;
-@grayLight1: #B8C4C2;
-@grayLight2: #E7EEEC;
-@grayLight3: #F9FBFA;
-@white: #ffffff;
-
-@greenDark3: #0E6834;
-@greenDark2: #116149;
-@greenBase: #13AA52;
-@greenLight2: #C3E7CA;
-@greenLight3: #E4F4E4;
-
-@blueDark3: #0D324F;
-@blueDark2: #1A567E;
-@blueBase: #007CAD;
-@blueLight2: #C5E4F2;
-@blueLight3: #E1F2F6;
-
-@yellowDark3: #543E07;
-@yellowDark2: #86681D;
-@yellowBase: #FFDD49;
-@yellowLight2: #FEF2C8;
-@yellowLight3: #FEF7E3;
-
-@redDark3: #570B08;
-@redDark2: #8F221B;
-@redBase: #CF4A22;
-@redLight2: #F9D3C5;
-@redLight3: #FCEBE2;
-
-@bluelight2: #C5E4F2;
-@bluebase: #007CAD;
-
-@purple0: #373e50;
-@purple1: #535d77;
-@purple2: #6e7c9f;
-@purple3: #8b96b2;
-@purple4: #a8b0c5;
-@purple5: #c5cbd9;
-
-/* Alert Colors */
-@alertOrange: #fbb129;
-@alertOrangeBg: #fff4dc;
-@alertOrangeBorder: #a06c29;
-@alertRed: #ef4c4c;
-@alertRedBg: #feeded;
-@alertRedText: #A94442;
-@alertBlue: #43b1e5;
-
-/* Semantic Colors */
-
-@inactiveIcon: @gray4;
-
+@palette__white: #ffffff;
```

Comment:
i'll move this to an import from LG in a separate PR, importing here directly from `@leafygreen-ui` would trigger the check since we are outside of compass-components